app/queries/items_out_query.rb

```diff
@@ -14,12 +14,13 @@ def initialize(organization:, storage_location:, filter_params: nil)
 
   # rubocop:disable Naming/MemoizedInstanceVariableName
   def call
-    @items ||=  LineItem.joins("
+    @items ||= LineItem.joins("
 LEFT OUTER JOIN distributions ON distributions.id = line_items.itemizable_id AND line_items.itemizable_type = 'Distribution'
 LEFT OUTER JOIN items ON items.id = line_items.item_id
 LEFT OUTER JOIN adjustments ON adjustments.id = line_items.itemizable_id AND line_items.itemizable_type = 'Adjustment'
-LEFT OUTER JOIN transfers ON transfers.id = line_items.itemizable_id AND line_items.itemizable_type = 'Transfer'")
-  .where("(distributions.storage_location_id = :id or (adjustments.storage_location_id= :id and line_items.quantity < 0) or transfers.from_id = :id) and items.organization_id= :organization_id",
+LEFT OUTER JOIN transfers ON transfers.id = line_items.itemizable_id AND line_items.itemizable_type = 'Transfer'
+LEFT OUTER JOIN kit_allocations ON kit_allocations.id = line_items.itemizable_id AND line_items.itemizable_type = 'KitAllocation' AND kit_allocations.kit_allocation_type = 'inventory_out'")
+  .where("(distributions.storage_location_id = :id or (adjustments.storage_location_id= :id and line_items.quantity < 0) or transfers.from_id = :id or kit_allocations.storage_location_id = :id) and items.organization_id= :organization_id",
```

Comment:
Wondering if we can extract some of this into a reusable scope / query on its own? Seems odd that it's repeated four times.

Reply:
@dorner added two scopes one for items_in and one for items_out since there it's different for both